File: Sources/PDFjet/SquareAnnotation.swift
/**
 * SquareAnnotation.swift
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */
import Foundation

/// A square annotation.
public class SquareAnnotation: BaseAnnotation {
    override public init() {
        super.init()
        self.annotationType = Annotation.Square
    }
}

File: com/pdfjet/SquareAnnotation.java
/*
 * SquareAnnotation.java
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */
package com.pdfjet;

/**
 * A square annotation.
 */
public class SquareAnnotation extends BaseAnnotation {
    /**
     * Creates a square annotation.
     */
    public SquareAnnotation() {
        super.annotationType = Annotation.Square;
    }
}
